src/Tools/Console/Command/MappingDescribeCommand.php

```diff
@@ -55,6 +56,7 @@ protected function configure(): void
              ->addArgument('entityName', InputArgument::REQUIRED, 'Full or partial name of entity')
              ->setDescription('Display information about mapped objects')
              ->addOption('em', null, InputOption::VALUE_REQUIRED, 'Name of the entity manager to operate on')
+            ->addOption('format', null, InputOption::VALUE_REQUIRED, 'Output format (json, text)', 'text')
```

Comment:
There seems to be an indention problem here.
```suggestion
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'Output format (json, text)', 'text')
```